test/benchmarks/benchmarks.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=missing-docstring,redefined-builtin
+
+import os
+
+from qiskit import QuantumCircuit
+from qiskit import execute
+from qiskit_jku_provider import QasmSimulator
+
+
+class QasmSimulatorJKUBenchmarkSuite:
+    """Runs the Basic qasm_simulator tests from Terra on JKU."""
+    params = ["grover_12.qasm", "sym9_193.qasm", "plus63mod4096_163.qasm", "z4_268.qasm",
+              "clip_206.qasm"]
```

Comment:
Doing it this way might cause some weird output on the dashboard, you'll be able to compare the performance of these parameter side by side (ie make the x-axis of the graphs each qasm circuit). That might be desirable I'm not sure, just something to think about.